OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -51,8 +51,20 @@ public void RenderAfterWorld(WorldRenderer wr, Actor self)
 			var iz = 1 / wr.Viewport.Zoom;
 
 			if (healthInfo != null)
+			{
 				healthInfo.Shape.DrawCombatOverlay(wr, wcr, self);
 
+				var targetZc = Color.Magenta;
+				var tx = new WVec(96, 0, 0);
```

Comment:
I don't mean that I'm generally against making it customizable, I just didn't bother while working on this PR.